fujiji-server/src/controllers/comment.js

```diff
@@ -37,4 +41,25 @@ async function postComment(req, res, next) {
   }
 }
 
-module.exports = { postComment };
+// PURPOSE: implement the get comments by listing id endpoint
+async function getListingComments(req, res, next) {
+  const { listing_id: listingID } = req.params;
+
+  const listing = await getListingById(parseInt(listingID, 10));
+
+  if (!listing) {
+    return next(
+      new ListingNotFound(`listing with id:${listingID} is not found`),
+    );
+  }
+
+  try {
+    const comments = await getComments(listingID);
+
+    return res.status(200).json({ comments });
+  } catch (err) {
+    return next(new APIError(err, 500));
+  }
```

Comment:
Can we also include these data from #103? I think we are missing the author's name and modified date.

Each object in the list should contains:
- The comment ID
- The author ID
- author's name
- text
- time stamp
- isAuthor
- modified date (last udpate)

I'm not sure if modified date column exist yet, though.

Reply:
Yes, it's the commenter's name. Can you also include the script for adding the column inside the migration-scripts directory? Thanks!

Reply:
For sure!

Reply:
Fixed!